src/lib/date-helper.js

```diff
@@ -53,7 +89,22 @@ function formatDateDefault(date) {
     'dateFormat'
   ]
   const timestamp = date instanceof Date ? date.getTime() : parseInt(date?.match(/\d+/)[0], 10)
-  const formattedDate = format(new Date(timestamp), formats)
+  const formattedDate = format(timeStamptoDate(timestamp), formats)
+
+  return formattedDate
+}
+
+function formatDateTimeDefault(date) {
+  if (!date) return
+
+  const formats = JSON.parse(window.localStorage.getItem('default') && window.localStorage.getItem('default'))[
+    'dateFormat'
+  ]
+
+  const timestamp = date instanceof Date ? date.getTime() : parseInt(date?.match(/\d+/)[0], 10)
+
+  const fullFormat = `${formats} hh:mm a`
```

Comment:
how can controll time format ex : 2:30 pm , 14: 30 or 14:30:20

Reply:
we only used the format we need which is hh:mm a in outbound
i can't understand your question well

Reply:
formatDateTimeDefault do we pass param format to it?

Reply:
postponed until needed